docs/user-guide/promql.md

```diff
@@ -0,0 +1,131 @@
+
+- Supported:
+    | Operator | Example  |
+    | :------- | :------- |
+    | add      | `a + b`  |
+    | sub      | `a - b`  |
+    | mul      | `a * b`  |
+    | div      | `a / b`  |
+    | mod      | `a % b`  |
+    | eqlc     | `a == b` |
+    | neq      | `a != b` |
+    | gtr      | `a > b`  |
+    | lss      | `a < b`  |
+    | gte      | `a >= b` |
+    | lte      | `a <= b` |
+
+- Unsupported:
+    | Operator | Progress |
+    | :------- | :------- |
+    | power    | TBD      |
+    | atan2    | TBD      |
+    | and      | TBD      |
+    | or       | TBD      |
+    | unless   | TBD      |
+
+### Aggregators
+
+- Supported:
+    | Aggregator | Example                   |
+    | :--------- | :------------------------ |
+    | sum        | `sum by (foo)(metric)`    |
+    | avg        | `avg by (foo)(metric)`    |
+    | min        | `min by (foo)(metric)`    |
+    | max        | `max by (foo)(metric)`    |
+    | stddev     | `stddev by (foo)(metric)` |
+
+- Unsupported:
+    | Aggregator   | Progress |
+    | :----------- | :------- |
+    | count        | TBD      |
+    | grouping     | TBD      |
+    | stdvar       | TBD      |
+    | topk         | TBD      |
+    | bottomk      | TBD      |
+    | count_values | TBD      |
+    | quantile     | TBD      |
+
+### Instant Functions
+
+- Supported:
+    | Function | Example         |
+    | :------- | :-------------- |
+    | abs      | `abs(metric)`   |
+    | ceil     | `ceil(metric)`  |
+    | exp      | `exp(metric)`   |
+    | ln       | `ln(metric)`    |
+    | log2     | `log2(metric)`  |
+    | log10    | `log10(metric)` |
+    | sqrt     | `sqrt(metric)`  |
+    | acos     | `acos(metric)`  |
+    | asin     | `asin(metric)`  |
+    | atan     | `atan(metric)`  |
+    | sin      | `sin(metric)`   |
+    | cos      | `cos(metric)`   |
+    | tan      | `tan(metric)`   |
+
+- Unsupported:
+    | Function                   | Progress |
+    | :------------------------- | :------- |
+    | absent                     | TBD      |
+    | scalar                     | TBD      |
+    | sgn                        | TBD      |
+    | sort                       | TBD      |
+    | sort_desc                  | TBD      |
+    | timestamp                  | TBD      |
+    | acosh                      | TBD      |
+    | asinh                      | TBD      |
+    | atanh                      | TBD      |
+    | sinh                       | TBD      |
+    | cosh                       | TBD      |
+    | tanh                       | TBD      |
+    | deg                        | TBD      |
+    | rad                        | TBD      |
+    | *other multiple input fns* | TBD      |
+
+### Range Functions
+
+- Supported:
+    | Function | Example                |
+    | :------- | :--------------------- |
+    | increase | `increase(metric[5m])` |
+    | irate    | `irate(metric[5m])`    |
+    | idelta   | `idelta(metric[5m])`   |
```

Comment:
Maybe we can just leave the supported functions here? Some functions have not been listed here, like: [`<aggregation>_over_time()`](https://prometheus.io/docs/prometheus/latest/querying/functions/#aggregation_over_time).

Reply:
Updated